<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>752</width>
    <height>560</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>752</width>
    <height>560</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>752</width>
    <height>560</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Maze</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/Images/icon.ico</normaloff>:/Images/icon.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="mainWidget">
   <property name="styleSheet">
    <string notr="true">background-color: #F0F0F0;</string>
   </property>
   <widget class="QFrame" name="menuFrame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>752</width>
      <height>48</height>
     </rect>
    </property>
    <property name="toolTipDuration">
     <number>0</number>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #FFFFFF;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <widget class="QLabel" name="timeLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>0</y>
       <width>50</width>
       <height>48</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>11</pointsize>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #353535;
</string>
     </property>
     <property name="text">
      <string>TIME:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="restartButton">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>0</y>
       <width>48</width>
       <height>48</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	image: url(:/images/restart_normal.png);
	border:0px;
}
QPushButton:hover
{
	image: url(:/images/restart_hover.png);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="trackButton">
     <property name="geometry">
      <rect>
       <x>642</x>
       <y>0</y>
       <width>48</width>
       <height>48</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{	
	image: url(:/images/find_normal.png);
	border: 0px;
}
QPushButton:hover
{
	image: url(:/images/find_hover.png);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="fogButton">
     <property name="geometry">
      <rect>
       <x>594</x>
       <y>0</y>
       <width>48</width>
       <height>48</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{	
	image: url(:/images/fog_normal.png);
	border: 0px;
}
QPushButton:hover
{
	image: url(:/images/fog_hover.png);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="timeValueLabel">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>0</y>
       <width>200</width>
       <height>48</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>11</pointsize>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #353535;
</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </widget>
   <widget class="SceneWidget" name="sceneWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>48</y>
      <width>752</width>
      <height>512</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>752</width>
      <height>512</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>512</width>
      <height>640</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SceneWidget</class>
   <extends>QWidget</extends>
   <header>views/widgets/scenewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
